src/improvements/tasks/sep/017-U16-opcm-upgrade-v400-base/README.md

```diff
@@ -61,9 +61,9 @@ where:
 
 The facilitator, or someone acting on behalf of the facilitator, must perform the following steps:
 
-1. Simulate this task from the `BaseNested` 2/2 by running `SIMULATE_WITHOUT_LEDGER=1 just --dotenv-path $(pwd)/.env --justfile ../../../nested.just simulate base_nested`
-2. In the terminal logs, directly above the Domain Hash and Message Hash, this will log the Safe Transaction Hash. See the [Simulation](#simulation) section for more details on how to do this step.
-3. Run `just new task ApproveSafeHash` to create a new task, and update the config file so this Safe Transaction Hash is used.
+1. Simulate this task from the `BaseNested` 2/2 by running `just simulate-stack sep 017-U16-opcm-upgrade-v400-base "[0x0000000000000000000000000000000000000000,0x0000000000000000000000000000000000000000,0x0000000000000000000000000000000000000000,0x0000000000000000000000000000000000000000,0x646132A1667ca7aD00d36616AFBA1A28116C770A]"`
+2. In the terminal logs, search for `017-U16-opcm-upgrade-v400-base` to find where the logs for this task begin. Directly above the Domain Hash and Message Hash, the Safe Transaction Hash will be logged. See the [Simulation](#simulation) section for more details on how to do this step.
+3. Run `just new task GnosisSafeApproveHash` to create a new task, and update the config file so this Safe Transaction Hash is used.
```

Comment:
I think we should provide guidance what number that task should be given.
Perhaps it should be something like 017A and 017B?